src/components/MyAccount/EditInputGroup.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Icon,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Stack,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { VscEyeClosed, VscEye } from "react-icons/vsc";
+const EditInputGroup = ({
+  label,
+  inputType,
+  setNewFirstName,
+  setNewLastName,
+  setPassword,
+  incorrectPswMsg,
+  weakPasswordMsg,
+  errorMessage,
+  setErrorMessage,
+  setWeakPasswordMsg,
+  setIncorrectPswMsg,
+  keyBoardEvent,
+}: {
+  label: string;
+  inputType: string;
+  setNewFirstName?: React.Dispatch<React.SetStateAction<string>>;
+  setNewLastName?: React.Dispatch<React.SetStateAction<string>>;
+  setPassword?: React.Dispatch<React.SetStateAction<string>>;
+  incorrectPswMsg?: string;
+  weakPasswordMsg?: string;
+  errorMessage?: string;
+  setErrorMessage?: React.Dispatch<React.SetStateAction<string>>;
+  setWeakPasswordMsg?: React.Dispatch<React.SetStateAction<string>>;
+  setIncorrectPswMsg?: React.Dispatch<React.SetStateAction<string>>;
+  keyBoardEvent: (event: any) => void;
+}) => {
+  const [show, setShow] = useState(false);
+  const visibleClick = () => setShow(!show);
+  return (
+    <FormControl>
+      <FormLabel
+        fontFamily="Inter"
+        fontWeight={700}
+        fontSize="16px"
+        color={incorrectPswMsg || weakPasswordMsg || errorMessage ? "#C13D46" : "#344C5C"}
+      >
+        {label}
+      </FormLabel>
+      {inputType === "name" && (
+        <Input
+          _focus={{ backgroundColor: "#F1F6FF" }}
+          borderColor="#7C9FDF"
+          fontWeight={700}
+          fontSize="16px"
+          marginBottom={10}
+          onChange={(e) => {
+            setNewFirstName && setNewFirstName(e.target.value);
```

Comment:
Why one input can change two variable？

Reply:
Because I do not want to write two same input and change the two variable separately  Actually, I have combine two input into one. And only one variable can be set in one time.

Reply:
You can build a common component to do that.

Reply:
Fix That